tutorials/plot_dipole_orientations.py

```diff
@@ -0,0 +1,196 @@
+"""
```

Comment:
I always do:

    # encoding: utf-8

Is there a preference for one format over the other? (I'm fine with whatever, just curious)